### ISSUE TITLE
Support exotic notations for angles in HSL

### ISSUE THREAD

**Issue by bartveneman** [omgovich/colord#40]
I know these are very unlikely patterns to run into, but apparently they exist, so support for them would be nice (but definitely not required):

```css
test-angles {
  color: hsl(.75turn, 60%, 70%);
  color: hsl(4.71239rad, 60%, 70%);
}
```

Source: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#hsl_syntax_variations

**Comment by omgovich** [comment]
Hey @bartveneman. Thanks for the issue!
Feels like supporting of those units is quite important thing — I'll update the parsers in a few days.

**Comment by omgovich** [comment]
Thanks for your support on Patreon BTW.
It means a lot to me!

**Comment by omgovich** [comment]
Hey! Just released v1.6 with the support for all CSS angle units included.
